CONTRIBUTING.md

```diff
@@ -1,62 +1,69 @@
 <h2 align="center">Contributing</h2>
 
-<p align="center">Contributing to Dark Reader can be in a variety of ways. Here, you can learn more about it. Thank you in advance for reading this <code>CONTRIBUTING.md</code>.</p>
+<p align="center">YOU can contribute to Dark Reader in a variety of ways!  Here, you can learn more.  Thank you in advance for reading this document.</p>
```

Comment:
I always felt so weird with reading documents that tries to get your attraction by capitalizing certain words and using `!` - Feels like a TV ad that was on a budget.

Reply:
> I always felt so weird with reading documents that tries to get your attraction by capitalizing certain words and using `!` - Feels like a TV ad that was on a budget.